compiler/luci/pass/src/PropagateConcatenationQparam.test.cpp

```diff
@@ -195,22 +203,11 @@ TEST(PropagateConcatenationQparam, propagate_concat_quantparam)
 TEST(PropagateConcatenationQparam, propagate_concat_quantparam_NEG)
 {
   // Check negative cases where qparam is not propagated
-  // (1) concat is not uint8-quantized
-  // (2) concat has fused activation function
+  // (1) concat has fused activation function
+  // (2) concat has fused activation function and input is const
 
   SimpleConcatGraph g;
 
-  // concat is not uint8-quantized
-  g.concat_node.dtype(loco::DataType::S8);
-  luci::propagate_concat_quantparam(&g.concat_node);
-  EXPECT_FLOAT_EQ(g.concat_node.quantparam()->scale[0], 3.14);
-  EXPECT_EQ(g.concat_node.quantparam()->zerop[0], 77);
-  EXPECT_FLOAT_EQ(g.input_1.quantparam()->scale[0], 1.0);
-  EXPECT_EQ(g.input_1.quantparam()->zerop[0], 1);
-  EXPECT_FLOAT_EQ(g.input_2.quantparam()->scale[0], 2.0);
-  EXPECT_EQ(g.input_2.quantparam()->zerop[0], 2);
-  g.concat_node.dtype(loco::DataType::U8);
-
```

Comment:
I removed this test case because `propagate_concat_quantparam` is now part of `QunatizeWithMinMaxPass`, so dtype of concat is always uint8 (I added `assert` for that condition in [here ](https://github.com/Samsung/ONE/pull/4219/files#diff-40cdd42b1975a4d2a2cc451a01dbebe9R712)).